tasks/sep/006-0-sc-guardian-updates/VALIDATION.md

```diff
@@ -0,0 +1,74 @@
+- All addresses (in section headers and storage values) match the provided name, using the Etherscan and Superchain Registry links provided. This validates the bytecode deployed at the addresses contains the correct logic.
+- All key values match the semantic meaning provided, which can be validated using the storage layout links provided.
+
+## State Overrides
+
+The following state overrides should be seen:
+
+### `0xa87675ebb9501C7baE8570a431c108C1577478Fa` (The Security Council Safe)
+
+Links:
+- [Etherscan](https://sepolia.etherscan.io/address/0xE75Cd021F520B160BF6b54D472Fa15e52aFe5aDD)
+
+Enables the simulation by setting the threshold to 1:
+
+- **Key:** `0x0000000000000000000000000000000000000000000000000000000000000004` <br/>
+  **Value:** `0x0000000000000000000000000000000000000000000000000000000000000001`
+
+## State Changes
+
+### `0x54e8baccc67fa3c6b3e9a94baa4d70d1668f0820` (`LivenessGuard`)
+
+Links:
+- [Etherscan](https://sepolia.etherscan.io/address/0x54e8baccc67fa3c6b3e9a94baa4d70d1668f0820) (not verified)
+
+State Changes:
+- **Key:** `0x023c107741c8297b7d31ce48158a7d3e0c1bab34eb099ba05b48e6e0bb5b0324` <br/>
+  **Before:** `0x000000000000000000000000000000000000000000000000000000006620aaa4` <br/>
+  **After:** `0x000000000000000000000000000000000000000000000000000000006621dcfe` <br/>
+  **Meaning:** The 'lastLive' timestamp of the caller has been updated. The key can be verified by:
+    1. Seeing that the [slot of the mapping is 0](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/snapshots/storageLayout/LivenessGuard.json#L6).
+    2. obtaining the storage key with `cast`, where the address provided should be the one used to simulate.
+      ```shell
+        cast index address 0x2e2e33fedd27fdecfc851ae98e45a5ecb76904fe 0
+        0x023c107741c8297b7d31ce48158a7d3e0c1bab34eb099ba05b48e6e0bb5b0324
+      ```
+
+- **Key**: `0x56ee16ca3ade18209faccff732edefbb77524a2f2c0c642df2abe4924871e783` <br/>
+  **Before**: `0x000000000000000000000000000000000000000000000000000000006620aaa4` <br/>
+  **After**: `0x0000000000000000000000000000000000000000000000000000000000000000` <br/>
+  **Meaning:** The 'lastLive' timestamp of the removed owner has been deleted. The key can be verified by:
+    1. Seeing that the [slot of the mapping is 0](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/snapshots/storageLayout/LivenessGuard.json#L6).
+    2. obtaining the storage key with `cast`, where the address provided should be the one used to simulate.
+      ```shell
+        cast index address 0x78339d822c23D943E4a2d4c3DD5408F66e6D662D 0
+        0x56ee16ca3ade18209faccff732edefbb77524a2f2c0c642df2abe4924871e783
+      ```
+
+### `0xa87675ebb9501c7bae8570a431c108c1577478fa` (`GnosisSafe`)
+
+Links:
+- [Etherscan](https://sepolia.etherscan.io/address/0xa87675ebb9501c7bae8570a431c108c1577478fa)
+
+State Changes:
```

Comment:
<img width="517" alt="Screenshot 2024-04-19 at 09 39 58" src="https://github.com/ethereum-optimism/superchain-ops/assets/15608778/aed551c5-bd79-43ce-b84b-824a7e4e0b21">

Below captures what's in this image but I don't know if we have strong convention here to keep the exact state changes in or not.

Reply:
We have always used that raw format in the past because Tenderly's decoding was broken, but I used this format because it seems to be fixed (for some contracts), and my [simulation has the values decoded](https://www.tdly.co/shared/simulation/b1a752b4-bab3-47be-adb3-2809b3d880af). 

Can you see this at that link? 

![2024-04-19 at 10 30@2x](https://github.com/ethereum-optimism/superchain-ops/assets/23033765/26b71591-74ef-427c-858e-4f25a6f7425e)

Reply:
Cool, our tenderly links are the exact same bar the live `lastLive` timestamp.